charmhelpers/contrib/openstack/templates/haproxy.cfg

```diff
@@ -88,9 +88,9 @@ backend {{ service }}_{{ frontend }}
     {% endif -%}
     {% for unit, address in frontends[frontend]['backends'].items() -%}
     {% if https -%}
-    server {{ unit }} {{ address }}:{{ ports[1] }} check check-ssl verify none
+    server {{ unit }} {{ address }}:{{ ports[1] }} check check-ssl verify none send-proxy-v2 check-send-proxy
```

Comment:
why are you using check-send-proxy only for https traffic?

https://docs.haproxy.org/2.5/configuration.html#check-send-proxy

Reply:
right I think I missed. I added it to for http as well. Thanks!